src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.ts

```diff
@@ -127,17 +128,186 @@ export class InstructorHelpQuestionsSectionComponent extends InstructorHelpSecti
 
   @Output() collapsePeerEvalTips: EventEmitter<any> = new EventEmitter();
 
-  constructor(private modalService: NgbModal,
+  constructor(private simpleModalService: SimpleModalService,
               private pageScrollService: PageScrollService,
               @Inject(DOCUMENT) private document: any) {
     super();
   }
 
   /**
-   * Opens modal window.
+   * Opens modal for contribution info.
    */
-  openModal(modal: any): void {
-    this.modalService.open(modal);
+  openContribInfoModal(): void {
+    const modalContent: string = `<div class="modal-body">
+      <ul>
```

Comment:
We're putting too much of HTML here. If there is this large amount of HTML code necessary, should keep the HTML code inside the HTML file and pass by reference to `TemplateRef`.

Implementation reference:
- https://github.com/TEAMMATES/teammates/blob/master/src/web/app/components/toast/toast.component.html
- https://github.com/TEAMMATES/teammates/blob/master/src/web/app/components/toast/toast.component.ts
- https://github.com/TEAMMATES/teammates/blob/master/src/web/app/components/toast/toast.ts

Usage example:
https://github.com/TEAMMATES/teammates/blob/1bbf28caa2bef26f3bc148a61ed03608eac9d35b/src/web/app/pages-instructor/instructor-courses-page/add-course-form/add-course-form.component.html#L47-L52
https://github.com/TEAMMATES/teammates/blob/1bbf28caa2bef26f3bc148a61ed03608eac9d35b/src/web/app/pages-instructor/instructor-courses-page/add-course-form/add-course-form.component.ts#L21
https://github.com/TEAMMATES/teammates/blob/1bbf28caa2bef26f3bc148a61ed03608eac9d35b/src/web/app/pages-instructor/instructor-courses-page/add-course-form/add-course-form.component.ts#L72